packages/components/src/components/hds/dropdown/toggle/icon.hbs

```diff
@@ -8,11 +8,16 @@
   ...attributes
   aria-expanded={{if @isOpen "true" "false"}}
   {{on "click" this.onClick}}
+  {{did-update this.onDidUpdateImageSrc @imageSrc}}
   type="button"
 >
   <div class="hds-dropdown-toggle-icon__wrapper">
     {{#if @imageSrc}}
-      <img src={{@imageSrc}} alt="" role="presentation" height="32" width="32" />
+      {{#if this.hasImage}}
+        <img src={{@imageSrc}} alt="" role="presentation" height="32" width="32" {{on "error" this.onImageLoadError}} />
```

Comment:
not necessarily in this PR, but should we remove the `width/height` attributes, since the size is set by CSS depending on the toggle `@size`?